Casks/m/mouseless.rb

```diff
@@ -4,7 +4,7 @@
 
   url "https://mouseless.click/mouseless-installer_v#{version}.dmg"
   name "mouseless"
-  desc "Mouse control with the keyboard"
+  desc "Full, fast mouse control with the keyboard"
```

Comment:
```suggestion
  desc "Mouse control with the keyboard"
```
Per our `desc` stanza guidelines, we trim the marketing fluff out of descriptions in `homebrew-cask`.
https://docs.brew.sh/Cask-Cookbook#stanza-desc

Reply:
understandable, thank you for the link, will follow those from now on.  I asked claude if it was appropriate, but apparently he's not up to date on those guidelines.